setup.cfg

```diff
@@ -31,5 +31,5 @@ console_scripts =
     snp-create-id-block = sevsnpmeasure.id_block:main
 
 [flake8]
-max-complexity = 10
+max-complexity = 12
```

Comment:
Please also change `Makefile` accordingly (this is what breaks the CI in github).

Reply:
Or remove the arguments from the call in Makefle and have it pick up the settings from setup.cfg?

Reply:
Yes, you're right, if it automatically reads setup.cfg. Your way is even better.